mission_control/navigator_missions/nav_missions/pinger.py

```diff
@@ -68,6 +77,7 @@ def get_observation_poses(self):
     @txros.util.cancellableInlineCallbacks
     def search_samples(self):
         """Move to each observation point and listen to the pinger while sitting still"""
+        yield self.get_observation_poses()
```

Comment:
(Assuming `self.observation_points` and `self.look_at_points` are only used here) Any reason not to do:
```python
obs, looks = self.get_observation_poses()  # returns [obs_points, look_at_points]
for obs, look in zip(obs, looks):
    ...
    ...
```

Setting acouple class variables here doesn't make sense if you're only using it in this one place right?